api/app/services/highlighter.py

```diff
@@ -65,8 +65,11 @@ def similarity_matrix(self, vector1, vector2):
         return (vector1.unsqueeze(1) * vector2.unsqueeze(0)).sum(-1)
 
     def highlight_paragraph(self, query_state, para_state,
-                            para_words) -> List[Tuple[int]]:
-        '''Returns the start and end positions of sentences that have the to'''
+                            para_words, original_paragraph) -> List[Tuple[int]]:
+        '''Returns the start and end character positions of highlighted sentences'''
+
+        if original_paragraph is None or original_paragraph == "":
```

Comment:
```suggestion
        if original_paragraph:
```

This handles both `None` and `''`